pom.xml

```diff
@@ -0,0 +1,111 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.testeditor</groupId>
+  <artifactId>testeditor-maven-plugin</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>maven-plugin</packaging>
+
+  <name>testeditor-maven-plugin Maven Plugin</name>
```

Comment:
In the regular test-editor projects we avoid setting explicit names. The project is then displayed as `groupId:artifactId`